src/pp-annotator/form/AnnotationForm.tsx

```diff
@@ -54,77 +63,187 @@ export default class AnnotationForm extends React.Component<
 
     return (
-      <div className="pp-widget">
-        <div className="pp-editor-head-bar">
-            <div className={"pp-editor-priority" + (priority=='NORMAL' ? ' selected' : '')}>
-                przypis
+        <div className="pp-widget">
+          <div className="pp-editor-head-bar">
+            <label className="priority-header"> Co dodajesz? </label>
+            <Popup
+                on="click"
+                hideOnScroll
+                trigger={<div className="priority-help"> <i className="help circle icon"></i> </div>}
+                flowing
+                hoverable
+            >
+              {/*TODO just an instruction stub*/}
+              <Grid centered divided columns={3}>
+                <Grid.Column textAlign='center'>
+                  <Header as='h4'>Niebieski przypis</Header>
+                </Grid.Column>
+                <Grid.Column textAlign='center'>
+                  <Header as='h4'>Żółty przypis</Header>
+                </Grid.Column>
+                <Grid.Column textAlign='center'>
+                  <Header as='h4'>Pomarańczowy przypis</Header>
+                </Grid.Column>
+              </Grid>
+            </Popup>
+            <br/>
+            {/*KG todo could probably be neater if done with sth like PriorityButton component*/}
+            <div className="priority-normal">
+              <button
+                  className={"pp-editor-priority" + (priority == AnnotationPriorities.NORMAL ? ' selected' : '')}
+                  onClick={() => this.setState({priority: AnnotationPriorities.NORMAL})}
+              >
+                dodatkowa informacja
+              </button>
             </div>
-          <div className={"pp-editor-priority" + (priority=='WARNING' ? ' selected' : '')}>
-                przypis
+            <div className="priority-warning">
+              <button
+                  className={"pp-editor-priority" + (priority == AnnotationPriorities.WARNING ? ' selected' : '')}
+                  onClick={() => this.setState({priority: AnnotationPriorities.WARNING})}
+              >
+                wyjaśnienie
+              </button>
             </div>
-          <div className={"pp-editor-priority" + (priority=='ALERT' ? ' selected' : '')}>
-                przypis
+            <div className="priority-alert">
+              <button
+                  className={"pp-editor-priority" + (priority == AnnotationPriorities.ALERT ? ' selected' : '')}
+                  onClick={() => this.setState({priority: AnnotationPriorities.ALERT})}
+              >
+                sprostowanie błędu
+              </button>
             </div>
 
-
-        </div>
-          <div className="pp-close">
-          <i>X</i>
-        </div>
-        <ul className="pp-listing">
-          <li className="annotator-item">
-            <textarea
+          </div>
+          <div
+              className="pp-close"
+              onClick={(e) => this.onCancel(e)}
```

Comment:
`onClick={this.props.onCancel}` should be enough

Reply:
done